susemanager-utils/susemanager-sls/src/beacons/reboot_info.py

```diff
@@ -45,7 +51,18 @@ def beacon(config):
     ret = []
 
     # pylint: disable-next=undefined-variable
-    result = __salt__["reboot_info.reboot_required"]()
+    try:
+        result = __salt__["reboot_info.reboot_required"]()
```

Comment:
@wweellddeerr @vzhestkov 
Can we make an optimization here, and if the flag is already set to true on the __context__ then we don't need to run the salt module again? After restart the context is cleaned, and it would start pooling it again. Does it make sense?

Reply:
ah good point. ignore it :)